crates/matrix-sdk-base/src/client.rs

```diff
@@ -893,6 +897,9 @@ impl BaseClient {
     }
 
     pub(crate) async fn apply_changes(&self, changes: &StateChanges) {
+        if changes.account_data.contains_key(&GlobalAccountDataEventType::IgnoredUserList) {
```

Comment:
Is this present only when there's an actual change in the ignore list?

Reply:
yup! both if it's an ignore, or unignore event